bin/test-instruction-logger/bin/test-instruction-logger

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env php
+<?php
+ini_set('display_errors', 1);
+ini_set('display_startup_errors', 1);
+error_reporting(E_ALL);
+// Make sure this script is being run over the PHP CLI.
+if ( 'cli' !== php_sapi_name() ) {
+	return;
+}
+
+define('WP_USE_THEMES', false);
+
+$wc_admin_root = __DIR__ . '/../../../';
+require_once $wc_admin_root . '/../../../wp-load.php';
```

Comment:
I get an error from this line because I don’t have WordPress installed in this path (I'm using Docker).

```
Fatal error: require_once(): Failed opening required '/Users/xxxxxx/Code/woocommerce-admin/bin/test-instruction-logger/bin/../../..//../../../wp-load.php' (include_path='.:') in /Users/xxxxxx/Code/woocommerce-admin/bin/test-instruction-logger/bin/test-instruction-logger on line 14
```

Is there anything depending on WordPress?

Reply:
Ah, I see you're using the `Requests` lib. It might just be better to replace it with vanilla `curl` functions to avoid the dependency on WP.

Reply:
@adrianduffell Interesting 🤔 , isn't that the default location for WordPress?

Yeah, I'm using the `Requests` for multi curl requests. Good idea, I'll remove the `Requests` dependency.

Reply:
@adrianduffell Removed `Requests` dependency in https://github.com/woocommerce/woocommerce-admin/pull/7499/commits/d4acb7b977436bd514405d141f7fdd92aadb5332

Reply:
@adrianduffell I also made a change how to determine which PR to include in https://github.com/woocommerce/woocommerce-admin/pull/7499/commits/65ce01fdbbfd62a8dc40c4ab4b2487a77d6d611d. It's checking `no release testing instructions` label suggested by Matt in p90Yrv-2re#comment-6127

I've updated `Detailed test instructions:` accordingly

Reply:
Looks good! Thanks for making these changes.